ui/src/app/shared/components/checkbox-filter/checkbox-filter.tsx

```diff
@@ -13,6 +13,7 @@ interface Props {
 export class CheckboxFilter extends React.Component<Props> {
     constructor(props: any) {
         super(props);
+        this.props.items.forEach(item => this.props.selected.push(item.name));
```

Comment:
mmm this is a bit of an anti-pattern, especially for a reusable component like this one (it's used in more than one place).

It would be better to just set the initial `selected` props from the parent component. That ends up going two parents up with the initial state [here](https://github.com/argoproj/argo-workflows/blob/9cb378342283c9ef9f2f3b999bec7cf10c8aab91/ui/src/app/cron-workflows/components/cron-workflow-list/cron-workflow-list.tsx#L38)

Reply:
Fixed in #11739